--- wanpipe/util/ft1/wanpipe_cfgft1.orig	2004-03-24 22:28:37.000000000 +0100
+++ wanpipe/util/ft1/wanpipe_cfgft1	2004-04-09 18:22:25.541668048 +0200
@@ -1625,7 +1625,7 @@
 GET_RC="cat $RC"
 TEMP="$WAN_HOME/temp.$$"
 WAN_DEVICE=$1
-META_CONF=$WAN_HOME/wanrouter.rc
+META_CONF=/etc/sysconfig/wanrouter
 WAN_LOAD=NO
 
 source $META_CONF
--- wanpipe/util/wancfg/lib/chdlc_lib.sh.orig	2003-07-18 22:18:36.000000000 +0200
+++ wanpipe/util/wancfg/lib/chdlc_lib.sh	2004-04-09 18:23:20.363333888 +0200
@@ -1,4 +1,4 @@
-#!/bins/sh
+#!/bin/bash
 
 function chdlc_interface_init () {
 	
--- wanpipe/util/wancfg/lib/fr_lib.sh.orig	2003-07-18 22:18:36.000000000 +0200
+++ wanpipe/util/wancfg/lib/fr_lib.sh	2004-04-09 18:23:36.286913136 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 function fr_interface_setup () {
 
--- wanpipe/util/wancfg/lib/help.sh.orig	2003-07-18 22:18:36.000000000 +0200
+++ wanpipe/util/wancfg/lib/help.sh	2004-04-09 18:24:12.509406480 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 #============================================================
 # start_menu_help
--- wanpipe/util/wancfg/lib/lib.sh.orig	2004-02-02 23:43:18.000000000 +0100
+++ wanpipe/util/wancfg/lib/lib.sh	2004-04-09 18:26:41.126813200 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 function setup_name_prefix()
 {
@@ -4152,7 +4152,7 @@
 			NMSK_IP[$num]=$value
 			#echo "NMSK_IP=${NMSK_IP[$num]}"
 			  ;;
-		*POINTOPOINT*)
+		*POINTOPOINT*|REMIP)
 			R_IP[$num]=$value
 			#echo "R_IP=${R_IP[$num]}"
 			  ;;
@@ -4896,7 +4896,7 @@
 DEVICE=${IF_NAME[$if_num]}
 IPADDR=${L_IP[$if_num]}
 NETMASK=${NMSK_IP[$if_num]}
-POINTOPOINT=${R_IP[$if_num]}
+REMIP=${R_IP[$if_num]}
 ONBOOT=yes
 EOM
 		
--- wanpipe/util/wancfg/lib/ppp_lib.sh.orig	2003-07-18 22:18:36.000000000 +0200
+++ wanpipe/util/wancfg/lib/ppp_lib.sh	2004-04-09 18:26:49.412553576 +0200
@@ -1,4 +1,4 @@
-#!/bins/sh
+#!/bin/bash
 
 function ppp_interface_setup () {
 
--- wanpipe/util/wancfg/lib/x25_lib.sh.orig	2003-07-18 22:18:36.000000000 +0200
+++ wanpipe/util/wancfg/lib/x25_lib.sh	2004-04-09 18:26:59.112079024 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 function x25_interface_setup () {
 
--- wanpipe/util/wancfg/wancfg.orig	2004-03-24 22:28:37.000000000 +0100
+++ wanpipe/util/wancfg/wancfg	2004-04-09 18:28:25.656922208 +0200
@@ -1234,7 +1234,7 @@
 # check_setup  
 #
 #  	Check that all libraries exit as well as 
-#       /etc/wanrouter.rc.  
+#       /etc/sysconfig/wanrouter
 #       Wanpipe Package must be installed before
 #       proceeding.
 #============================================================
@@ -1277,12 +1277,12 @@
 		exit 1
 	fi
 
-	if [ ! -f "$PROD_HOME/wanrouter.rc" ]; then
+	if [ ! -f "/etc/sysconfig/wanrouter" ]; then
 		cat << EOM
 
 	BAD CONFIGURATION - FILE MISSING
 
-	$PROD_HOME/wanrouter.rc file could not 
+	/etc/sysconfig/wanrouter file could not 
 be found. Please make sure that the wanpipe
 pacakge has been correctly installed before
 proceeding to configure a Sangoma device.
@@ -1405,7 +1405,7 @@
 check_setup 1
 
 #Load the library files
-source $PROD_HOME/wanrouter.rc
+source /etc/sysconfig/wanrouter
 
 check_setup 2
 
