--- wanpipe/Setup.org	2006-09-17 14:54:57.778046000 +0000
+++ wanpipe/Setup	2006-09-17 14:57:37.596034000 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Setup		WANPIPE WAN Router Installation/Removal Script.
 #
@@ -3139,15 +3139,6 @@
 		CONFIG_SMP=YES;  
 	else	
 		echo -e "Disabled.\n"
-		eval "uname -a |grep SMP > /dev/null 2> /dev/null"
-		if [ $? -eq 0 ]; then
-			echo
-			echo "   Warning: Current image is configured for SMP"
-			getyn "  Would you like to proceed with installation?"
-			if [ $? -ne 0 ]; then
-				exit 1
-			fi 
-		fi
 		CONFIG_SMP=NO;
 	fi
 
