--- wanpipe-7.0.0/util/misc/Makefile.orig	2008-08-01 15:00:00.000000000 +0200
+++ wanpipe-7.0.0/util/misc/Makefile	2013-03-10 19:56:46.015102862 +0100
@@ -20,7 +20,8 @@
 endif
 
 # Tools options.
-CFLAGS  = -Wall -O2 -D$(OS_TYPE) -D_DEBUG_=$(DEBUG) -I$(SYSINC) -I$(WANINCDIR) $(HOSTCFLAGS)
+OPTFLAGS = -O2
+CFLAGS  = -Wall $(OPTFLAGS) -D$(OS_TYPE) -D_DEBUG_=$(DEBUG) -I$(SYSINC) -I$(WANINCDIR) $(HOSTCFLAGS)
 
 ####### RULES ################################################################
 
--- wanpipe-7.0.0/util/wanconfig/Makefile.orig	2008-08-01 15:00:00.000000000 +0200
+++ wanpipe-7.0.0/util/wanconfig/Makefile	2013-03-10 19:55:12.731773439 +0100
@@ -23,7 +23,8 @@
 
 # Tools options.
 SRCS=wanconfig.c wanconfig_hwec.c  
-CFLAGS = -Wall -O2 -D$(OS_TYPE) -I$(SYSINC) -D_DEBUG_=$(DEBUG)
+OPTFLAGS=-O2
+CFLAGS = -Wall $(OPTFLAGS) -D$(OS_TYPE) -I$(SYSINC) -D_DEBUG_=$(DEBUG)
 CFLAGS += -DCONFIG_PRODUCT_WANPIPE_AFT_BRI -DCONFIG_PRODUCT_WANPIPE_USB -I../libsangoma/wanec
 CFLAGS += -I../../patches/kdrivers/include -I$(SYSINC)
 CFLAGS += -I$(WANECDIR)/oct6100_api/include
--- wanpipe-7.0.5/util/wanpipemon/Makefile.orig	2013-07-21 20:28:26.970321129 +0200
+++ wanpipe-7.0.5/util/wanpipemon/Makefile	2013-07-21 20:29:16.680319043 +0200
@@ -34,11 +34,12 @@
 
 CC = gcc 
 CPP = gcc -E
+OPTFLAGS = -O2
 CFLAGS = -Wall  -fno-strict-aliasing -Wstrict-prototypes 
 CFLAGS += $(EXTRA_FLAGS) $(PROTOCOL_DEFS)
 CFLAGS += $(GLOBAL_CFLAGS)
 CFLAGS +=  -D$(OS_TYPE) -I../libsangoma -I../libsangoma/wanec -I$(WLINC) -I$(WEC) -I$(WECOCT) -I$(WINC)
-CFLAGS += -I$(SYSINC) -I$(SYSINC)/oct6100_api -I$(SYSINC)/oct6100_api/oct6100api -I../ft1 -O2 -fomit-frame-pointer -DLOCALE -D_DEBUG_=$(DEBUG)
+CFLAGS += -I$(SYSINC) -I$(SYSINC)/oct6100_api -I$(SYSINC)/oct6100_api/oct6100api -I../ft1 -fomit-frame-pointer $(OPTFLAGS) -DLOCALE -D_DEBUG_=$(DEBUG)
 WAN_VIRTUAL=
 
 
--- wanpipe-7.0.0/util/sdladump/Makefile.orig	2008-08-01 15:00:00.000000000 +0200
+++ wanpipe-7.0.0/util/sdladump/Makefile	2013-03-10 19:57:59.671766435 +0100
@@ -15,7 +15,8 @@
 endif
 
 # Tools options.
-CFLAGS  = -D__LINUX__ -I$(SYSINC) -Wall -O2  -D_DEBUG_=$(DEBUG) $(EXTRA_FLAGS) 
+OPTFLAGS = -O2
+CFLAGS  = -D__LINUX__ -I$(SYSINC) -Wall $(OPTFLAGS)  -D_DEBUG_=$(DEBUG) $(EXTRA_FLAGS) 
 
 
 ####### RULES ################################################################
--- wanpipe-7.0.0/util/ft1/Makefile.orig	2008-08-01 15:00:00.000000000 +0200
+++ wanpipe-7.0.0/util/ft1/Makefile	2013-03-10 19:58:40.768431374 +0100
@@ -22,7 +22,8 @@
 EXTRA_FLAGS=
 
 # Tools options.
-CFLAGS  = -D__LINUX__ -Wall -O2 -D_GNUC_ -I$(WANINCDIR) -I$(SYSINC) -D$(OS_TYPE) -D_DEBUG_=$(DEBUG)  
+OPTFLAGS = -O2
+CFLAGS  = -D__LINUX__ -Wall $(OPTFLAGS) -D_GNUC_ -I$(WANINCDIR) -I$(SYSINC) -D$(OS_TYPE) -D_DEBUG_=$(DEBUG)  
 LIBS = -lncurses
 
 CFLAGS += $(EXTRA_FLAGS)
--- wanpipe-7.0.0/util/bwm/Makefile.orig	2008-08-01 15:00:00.000000000 +0200
+++ wanpipe-7.0.0/util/bwm/Makefile	2013-03-10 19:59:10.858430099 +0100
@@ -1,5 +1,7 @@
+OPTFLAGS=-O2
+
 all: bwm.c
-	$(CC) $(CFLAGS) -O2 -Wall bwm.c -o wpbwm
+	$(CC) $(CFLAGS) $(OPTFLAGS) -Wall bwm.c -o wpbwm
 
 clean:
 	rm -f wpbwm
--- wanpipe-7.0.0/util/lxdialog/Makefile.orig	2013-03-10 19:10:00.295220605 +0100
+++ wanpipe-7.0.0/util/lxdialog/Makefile	2013-03-10 20:00:40.318426412 +0100
@@ -3,7 +3,8 @@
 #Will be overwritten by ./Setup
 SYSINC=/lib/modules/$(shell uname -r)/build/include
 
-CFLAGS = -D__LINUX__ -Wall -Wstrict-prototypes -I$(SYSINC) -O2 -fomit-frame-pointer -DLOCALE 
+OPTFLAGS = -O2
+CFLAGS = -D__LINUX__ -Wall -Wstrict-prototypes -I$(SYSINC) -fomit-frame-pointer $(OPTFLAGS) -DLOCALE 
 LDFLAGS = -s -L .
 LDLIBS = -lncurses
 WAN_VIRTUAL=
