--- wanpipe-3.5.20/patches/kdrivers/src/wanrouter/wanmain.c~	2011-04-17 15:15:10.162513484 +0200
+++ wanpipe-3.5.20/patches/kdrivers/src/wanrouter/wanmain.c	2011-04-17 15:15:24.459179809 +0200
@@ -2136,10 +2136,7 @@
 void wan_skb_destructor (struct sk_buff *skb)
 {
 	if (skb->dev){
-		struct net_device *dev=skb->dev;
-		atomic_dec(&(dev)->refcnt);
-		//printk(KERN_INFO "%s: Skb destructor: put dev: refcnt=%i\n",
-		//		dev->name,atomic_read(&dev->refcnt));
+		dev_put( skb->dev );
 	}	
 }
 
