diff -Naur wanrouter/api/Makefile wanrouter-p/api/Makefile
--- wanrouter/api/Makefile	Thu Apr  6 23:56:33 2000
+++ wanrouter-p/api/Makefile	Sat Jun  2 17:14:01 2001
@@ -17,7 +17,7 @@
 VPATH  = $(SYSINC)
 
 # Tools options.
-CFLAGS  = -Wall -O2 -D_GNUC_ -D$(OS_TYPE) -D_DEBUG_=$(DEBUG)  -I/usr/src/linux/include
+CFLAGS  = -Wall -O2 -D_GNUC_ -D$(OS_TYPE) -D_DEBUG_=$(DEBUG)  -I/usr/src/linux/include -I/usr/include/ncurses
 LIBS = -lncurses
 
 ####### RULES ################################################################
diff -Naur wanrouter/config/Makefile wanrouter-p/config/Makefile
--- wanrouter/config/Makefile	Thu Apr  6 23:56:32 2000
+++ wanrouter-p/config/Makefile	Sat Jun  2 17:13:45 2001
@@ -17,7 +17,7 @@
 VPATH  = $(SYSINC)
 
 # Tools options.
-CFLAGS  = -Wall -O2 -D_GNUC_ -D$(OS_TYPE) -D_DEBUG_=$(DEBUG)  -I/usr/src/linux/include
+CFLAGS  = -Wall -O2 -D_GNUC_ -D$(OS_TYPE) -D_DEBUG_=$(DEBUG)  -I/usr/src/linux/include -I/usr/include/ncurses
 LIBS = -lncurses
 
 ####### RULES ################################################################
diff -Naur wanrouter/config/ft1/Makefile wanrouter-p/config/ft1/Makefile
--- wanrouter/config/ft1/Makefile	Thu Apr  6 23:56:32 2000
+++ wanrouter-p/config/ft1/Makefile	Sat Jun  2 17:13:16 2001
@@ -17,7 +17,7 @@
 VPATH  = $(SYSINC)
 
 # Tools options.
-CFLAGS  = -Wall -O2 -D_GNUC_ -D$(OS_TYPE) -D_DEBUG_=$(DEBUG)  -I/usr/src/linux/include
+CFLAGS  = -Wall -O2 -D_GNUC_ -D$(OS_TYPE) -D_DEBUG_=$(DEBUG)  -I/usr/src/linux/include -I/usr/include/ncurses
 LIBS = -lncurses
 
 ####### RULES ################################################################
diff -Naur wanrouter/config/ft1/source/Makefile wanrouter-p/config/ft1/source/Makefile
--- wanrouter/config/ft1/source/Makefile	Sat Jun  2 17:12:13 2001
+++ wanrouter-p/config/ft1/source/Makefile	Sat Jun  2 17:13:14 2001
@@ -15,7 +15,7 @@
 VPATH  = $(SYSINC)
 
 # Tools options.
-CFLAGS  = -Wall $(OPTFLAGS) -D_GNUC_ -D$(OS_TYPE) -D_DEBUG_=$(DEBUG) -I/usr/src/linux/include
+CFLAGS  = -Wall $(OPTFLAGS) -D_GNUC_ -D$(OS_TYPE) -D_DEBUG_=$(DEBUG) -I/usr/src/linux/include -I/usr/include/ncurses
 LIBS = -lncurses
 
 ####### RULES ################################################################
