--- wanpipe/Setup.org	2006-09-17 14:54:57.778046000 +0000
+++ wanpipe/Setup	2006-09-17 14:57:37.596034000 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Setup		WANPIPE WAN Router Installation/Removal Script.
 #
@@ -2921,15 +2921,6 @@
 		CONFIG_SMP=YES;
 	else	
 		echo -e "Disabled.\n"
-		eval "uname -a |grep SMP > /dev/null 2> /dev/null"
-		if [ $? -eq 0 ]; then
-			echo
-			echo "   Warning: Current image is configured for SMP"
-			getyn "   Would you like to cancel installation?"
-			if [ $? -eq 0 ]; then
-				exit 1
-			fi 
-		fi
 		CONFIG_SMP=NO;
 	fi
 
@@ -5574,11 +5565,6 @@
 fi
 LINUXDRIVERS_WAN="net/wanrouter"
 
-[ "$UID" = 0 ] || {
-	error "You must be a superuser (root) to run this script"
-	exit 1
-}
-
 # Check router home directory.
 [ -d "$PROD_HOME" ] || {
 	error "Router home directory ($PROD_HOME) not found"
