--- wanpipe/util/ft1/wanpipe_cfgft1.orig	2004-03-24 22:28:37.000000000 +0100
+++ wanpipe/util/ft1/wanpipe_cfgft1	2004-04-09 18:22:25.541668048 +0200
@@ -1625,7 +1625,7 @@
 GET_RC="cat $RC"
 TEMP="$WAN_HOME/temp.$$"
 WAN_DEVICE=$1
-META_CONF=$WAN_HOME/wanrouter.rc
+META_CONF=/etc/sysconfig/wanrouter
 WAN_LOAD=NO
 
 source $META_CONF
 #============================================================
 # start_menu_help
 
