--- wanpipe/util/misc/Makefile.orig	2003-07-18 22:18:36.000000000 +0200
+++ wanpipe/util/misc/Makefile	2004-04-09 18:39:26.032529856 +0200
@@ -15,7 +15,8 @@
 WAN_VIRTUAL=
 
 # Tools options.
-CFLAGS  = -Wall -O2 -D$(OS_TYPE) -D_DEBUG_=$(DEBUG) -I$(SYSINC) $(HOSTCFLAGS)
+OPTFLAGS=-O2
+CFLAGS  = -Wall $(OPTFLAGS) -D$(OS_TYPE) -D_DEBUG_=$(DEBUG) -I$(SYSINC) $(HOSTCFLAGS)
 
 
 ####### RULES ################################################################
--- wanpipe/util/wanconfig/Makefile.orig	2003-07-18 22:18:35.000000000 +0200
+++ wanpipe/util/wanconfig/Makefile	2004-04-09 18:40:07.506224896 +0200
@@ -15,7 +15,8 @@
 WAN_VIRTUAL=
 
 # Tools options.
-CFLAGS  = -Wall -O2 -D$(OS_TYPE) -I$(SYSINC) -D_DEBUG_=$(DEBUG)
+OPTFLAGS=-O2
+CFLAGS  = -Wall $(OPTFLAGS) -D$(OS_TYPE) -I/usr/include/ncurses -I../../patches/kdrivers/include -D_DEBUG_=$(DEBUG)
 
 ####### RULES ################################################################
 
--- wanpipe/util/wanpipemon/Makefile.orig	2004-01-15 17:10:28.000000000 +0100
+++ wanpipe/util/wanpipemon/Makefile	2004-04-09 18:41:11.266531856 +0200
@@ -12,8 +12,6 @@
 
 PROD=wanpipemon
 
-SYSINC=$(shell echo "/lib/modules/$(shell uname -r)/build/include")
-
 WAN_VIRTUAL=
 LXDIR=../lxdialog
 ENABLE_GUI=YES
@@ -23,11 +21,14 @@
 ASTBROOT=/usr
 #-DCONFIG_PRODUCT_WANPIPE_ADSL
 
+HOSTCFLAGS=-I../../patches/kdrivers/include -I/usr/include/ncurses
 CC = gcc 
 CPP = gcc -E
 CFLAGS = -Wall -Wstrict-prototypes 
 CFLAGS += $(EXTRA_FLAGS)
-CFLAGS += -I$(SYSINC) -D$(OS_TYPE) -I../ft1 -O2 -fomit-frame-pointer -DLOCALE -D_DEBUG_=$(DEBUG)
+OPTFLAGS=-O2
+CFLAGS =  $(OPTFLAGS) -fomit-frame-pointer -DLOCALE -D$(OS_TYPE) -I../ft1 -D_DEBUG_=$(DEBUG) $(HOSTCFLAGS)
+
 WAN_VIRTUAL=
 
 
--- wanpipe/util/sdladump/Makefile.orig	2003-07-18 22:18:35.000000000 +0200
+++ wanpipe/util/sdladump/Makefile	2004-04-09 18:39:45.744533176 +0200
@@ -13,7 +13,8 @@
 SYSINC	= /usr/src/linux/include
 
 # Tools options.
-CFLAGS  = -D__LINUX__ -I$(SYSINC) -Wall -O2  -D_DEBUG_=$(DEBUG) 
+OPTFLAGS=-O2
+CFLAGS  = -D__LINUX__ -I/usr/include/ncurses -I../../patches/kdrivers/include -Wall $(OPTFLAGS)  -D_DEBUG_=$(DEBUG) 
 
 
 ####### RULES ################################################################
--- wanpipe/util/ft1/Makefile.orig	2003-07-18 22:18:36.000000000 +0200
+++ wanpipe/util/ft1/Makefile	2004-04-09 18:38:37.559898816 +0200
@@ -11,10 +11,12 @@
 DEBUG	= 2
 
 # Project file paths.
-SYSINC=/lib/modules/$(shell uname -r)/build/include
+HOSTCFLAGS=-I../../patches/kdrivers/include -I/usr/include/ncurses
 
 # Tools options.
-CFLAGS  = -D__LINUX__ -Wall -O2 -D_GNUC_ -I$(SYSINC) -D$(OS_TYPE) -D_DEBUG_=$(DEBUG)  
+OPTFLAGS=-O2
+CFLAGS  = -D__LINUX__ -Wall $(OPTFLAGS) -D_GNUC_ -D$(OS_TYPE) -D_DEBUG_=$(DEBUG) $(HOSTCFLAGS)
+
 LIBS = -lncurses
 
 ####### RULES ################################################################
--- wanpipe/util/bwm/Makefile.orig	2003-10-08 17:38:05.000000000 +0200
+++ wanpipe/util/bwm/Makefile	2004-04-09 18:38:09.166215312 +0200
@@ -1,5 +1,8 @@
+CC=gcc
+OPTFLAGS=-O2
+
 all: bwm.c
-	$(CC) $(CFLAGS) -O2 -Wall bwm.c -o wpbwm
+	$(CC) $(CFLAGS) $(OPTFLAGS) -Wall bwm.c -o wpbwm
 
 clean:
 	rm wpbwm
--- wanpipe/util/lxdialog/Makefile.orig	2003-07-18 22:18:36.000000000 +0200
+++ wanpipe/util/lxdialog/Makefile	2004-04-09 18:39:10.810843904 +0200
@@ -1,15 +1,14 @@
 CC = gcc 
 CPP = gcc -E
 #Will be overwritten by ./Setup
-SYSINC=/lib/modules/$(shell uname -r)/build/include
 
-CFLAGS = -Wall -Wstrict-prototypes -I$(SYSINC) -O2 -fomit-frame-pointer -DLOCALE 
+OPTFLAGS=-O2 -fomit-frame-pointer                                              
+CFLAGS = -Wall -Wstrict-prototypes $(OPTFLAGS) -DLOCALE -I/usr/include/ncurses
 LDFLAGS = -s -L .
 LDLIBS = -lncurses
 WAN_VIRTUAL=
 
 CFLAGS += -I$(ASTBROOT)/include -DCURSES_LOC="<ncurses.h>"
-
 #ifeq (/usr/include/ncurses/ncurses.h, $(wildcard /usr/include/ncurses/ncurses.h))
 #        CFLAGS += -I/usr/include/ncurses -DCURSES_LOC="<ncurses.h>"
 #else
--- wanpipe/util/Makefile.org	2006-06-10 22:56:52.165416750 +0000
+++ wanpipe/util/Makefile	2006-06-10 22:56:59.481874000 +0000
@@ -1,5 +1,5 @@
 PWD=$(shell pwd)
-SYSINC=$(PWD)/../patches/kdrivers/include
+SYSINC="../../patches/kdrivers/include -I/usr/include/ncurses"
 KDIR=/lib/modules/$(shell uname -r)/build
 
 WAN_VIRTUAL=$(PREFIX)
