--- wanpipe-3.5.8/Setup.org	2009-12-13 15:36:22.000000000 +0000
+++ wanpipe-3.5.8/Setup	2009-12-13 15:38:35.000000000 +0000
@@ -7886,7 +7886,7 @@
 				echo "Error: Zaptel source directory not found: $ZAPTEL_INSTALL_DIR";
 				exit 1;
 			fi
-			if [ ! -f "$ZAPTEL_INSTALL_DIR/Makefile" ]; then
+			if [ ! -f "$ZAPTEL_INSTALL_DIR/include/dahdi/dahdi_config.h" ]; then
 				echo "Error: Invalid zaptel source found in $ZAPTEL_INSTALL_DIR";
 				exit 1;
 			fi
