--- wanpipe-7.0.10/patches/kdrivers/src/wanrouter/af_wanpipe.c.orig	2014-02-16 08:53:50.845662793 +0100
+++ wanpipe-7.0.10/patches/kdrivers/src/wanrouter/af_wanpipe.c	2014-02-16 08:51:49.382334557 +0100
@@ -1736,7 +1736,7 @@
 	}	
 
 	read_lock(&wanpipe_sklist_lock);
-#if defined(LINUX_3_0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,9,0)
 	sk_for_each(sk, &wanpipe_sklist) {
 #elif defined(LINUX_2_6)
 	sk_for_each(sk, node, &wanpipe_sklist) {
--- wanpipe-7.0.10/patches/kdrivers/src/wanrouter/af_wanpipe_datascope.c.orig	2008-08-01 15:00:00.000000000 +0200
+++ wanpipe-7.0.10/patches/kdrivers/src/wanrouter/af_wanpipe_datascope.c	2014-02-16 08:51:26.309002194 +0100
@@ -152,7 +152,7 @@
 #ifdef LINUX_2_6
 	{
 	struct hlist_node *node;
-#ifdef LINUX_3_0
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,9,0)
 	sk_for_each(parent_sk, &wanpipe_parent_sklist) {
 #else
 	sk_for_each(parent_sk, node, &wanpipe_parent_sklist) {
